apps/snitch_core/test/data/schema/property_test.exs

```diff
@@ -0,0 +1,78 @@
+defmodule Snitch.Data.Schema.PropertyTest do
+  use ExUnit.Case
+  use Snitch.DataCase
+
+  import Ecto.Changeset
+
+  alias Snitch.Data.Schema.Property
+
+  @valid_attrs %{name: "Milk", display_name: "Amul milk"}
```

Comment:
@Jeshan7 let's use e-commerce property names for this.

Reply:
@ashish173 Okay